include/linux/vfio.h

```diff
@@ -48,11 +60,15 @@ struct vfio_device_ops {
 extern struct iommu_group *vfio_iommu_group_get(struct device *dev);
 extern void vfio_iommu_group_put(struct iommu_group *group, struct device *dev);
 
+void vfio_init_group_dev(struct vfio_device *device, struct device *dev,
```

Comment:
maybe we shouldn't export new functions in this commit since they are static here and not used by others.

Reply:
This commit creates the exports and the next commit uses them, it is a normal pattern to break up patches